db_upgrade_scripts/mosip_ida/sql/1.2.0.1-B2_to_1.2.0.1-B3_upgrade.sql

```diff
@@ -1,4 +1,14 @@
-\c mosip_ida
+-- -------------------------------------------------------------------------------------------------
+-- Database Name	: mosip_ida
+-- Release Version 	: 1.2
+-- Purpose    		: Database Alter scripts for the release for ID Authentication DB.       
+-- Create By   		: Ram Bhatt
+-- Created Date		: Apr-2021
+-- 
+-- Modified Date        Modified By         Comments / Remarks
+-- -------------------------------------------------------------------------------------------------
+----------------------------------------------------------------------------------------------------
+\c mosip_ida sysadmin
```

Comment:
we don't sysadmin, since upgrade will be done by postgres user, can you remove the sysadmin alone ?